pom.xml

```diff
@@ -51,7 +51,7 @@
     <maven.deploy.plugin.version>2.8.2</maven.deploy.plugin.version>
 
     <!--Backend properties-->
-    <ddf.version>2.25.1</ddf.version>
+    <ddf.version>2.26.0</ddf.version>
```

Comment:
There are couple other version upgrades in the DDF bump from 2.25.1 to 2.26.0.  Need to make sure they get updated here too.  For example: karaf.version = 4.2.9, spark.version = 2.9.2

Reply:
yeah I'm working on those now